src/components/ItemImage/ItemImage.tsx

```diff
@@ -18,9 +18,15 @@ export default class ItemImage extends React.PureComponent<Props> {
     const { className, item, src, hasBadge, badgeSize } = this.props
     const isSmart = getItemMetadataType(item) === ItemMetadataType.SMART_WEARABLE
 
+    const shouldChangeImage = ['3491d5dd-7c0d-4287-8635-0eecf6957aec', 'b0d65d16-241e-4cd5-8468-ae8464b5869a'].includes(item.id)
+
     return (
       <div className={`ItemImage is-image image-wrapper ${className}`} style={getBackgroundStyle(item.rarity)}>
-        <img className="image" src={src || getThumbnailURL(item)} alt={item.name} />
+        <img
+          className="item-image"
+          src={shouldChangeImage ? 'https://i.stack.imgur.com/ek3Yc.png' : src || getThumbnailURL(item)}
+          alt={item.name}
+        />
```

Comment:
You've left the testing images here hahaha.

Reply:
what a dork this programmer is